packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -272,32 +272,34 @@ class HttpRequestCookiesView extends StatelessWidget {
     final responseCookies = data.responseCookies;
     return LayoutBuilder(
       builder: (context, constraints) {
-        return Column(
-          children: [
-            if (responseCookies.isNotEmpty)
-              _buildCookiesTable(
-                context,
-                'Response Cookies',
-                responseCookies,
-                constraints,
-                responseCookiesKey,
-              ),
-            // Add padding between the cookie tables if displaying both
-            // response and request cookies.
-            if (responseCookies.isNotEmpty && requestCookies.isNotEmpty)
-              const Padding(
-                padding: EdgeInsets.only(bottom: 24.0),
-              ),
-            if (requestCookies.isNotEmpty)
-              _buildCookiesTable(
-                context,
-                'Request Cookies',
-                requestCookies,
-                constraints,
-                requestCookiesKey,
-                requestCookies: true,
-              ),
-          ],
+        return SingleChildScrollView(
+          child: Column(
```

Comment:
Why not a ListView instead of a Column and a SingleChildScrollView?
You can't be getting any of the benefits of the column once it is wrapped in a scroll view because you would get a constraint assert error as soon as any child used a flex factor as the height is unconstrained.

Reply:
done